content/en/docs/concepts/configuration/secret.md

```diff
@@ -270,7 +270,7 @@ should be specified in a `kustomization.yaml` inside a directory.
 
 For example, to generate a Secret from files `./username.txt` and `./password.txt`
 ```shell
-# Create a kustomization.yaml file with SecretGenerator
+# Create a kustomization.yaml file with secretGenerator
 cat <<EOF >./kustomization.yaml
```

Comment:
Hello @PabloCastellano . I _think_ lines 273 and 312 generally refer to the actual generator (SecretGenerator) not the field in the kustomization.yaml. The wording could be improved.
->Create a kustomization.yaml file that uses a SecretGenerator OR
->Create a kustomization.yaml file which includes a secretGenerator field

See https://github.com/kubernetes-sigs/kustomize/blob/master/api/builtins/secretgenerator.go

Reply:
Hi @PabloCastellano . I reread the content. Some suggestions for improving clarity.
- What do you think about removing the comments from the YAML examples (lines 273 and 312)?
- Fix/Edit line 271:
For example, to generate a Secret from files `./username.txt` and `./password.txt`, 
you can set a `secretGenerator` in a `kustomization.yaml`:
- Fix/Edit Line 309:
For example, to generate a Secret from literals `username=admin` and `password=secret`,
you can specify a `secretGenerator` in a `kustomization.yaml`:

Reply:
LGTM, will submit the changes shortly

Reply:
Done

Reply:
Since I did not hear back from you for some time, I created the updates in, PR #18320. Let me know what you think.

Reply:
@kbhawkey Looks good, I'm closing this issue in favor of yours!